lib/transforms/multipart.ts

```diff
@@ -45,7 +45,7 @@ function mixedParser(ct: MultipartContentType): (reader: Reader<Buffer>, writer:
             if (i < 0) throw new Error('boundary not found');
             const lines = str.substring(0, i).split(/\r?\n/);
             const headers = lines.slice(0, lines.length - 2).reduce((h: any, l: string) => {
-                const kv = l.split(/\s*:\s*/);
+                const kv = Buffer.from(l, 'binary').toString('utf8').split(/\s*:\s*/);
```

Comment:
You should isolate this conversion into a function. I would call it `latin1ToUtf8(l)`

Reply:
done